vignettes/hubverse_ms_vignette.Rmd

```diff
@@ -0,0 +1,220 @@
+
+```{r, message=FALSE}
+target_data <-
+  aws.s3::s3read_using(
+    readr::read_csv,
+    object = "s3://cdcepi-flusight-forecast-hub/target-data/target-hospital-admissions.csv" #nolint
+  ) |>
+  dplyr::mutate(target = "wk inc flu hosp") |>
+  dplyr::select(date, target, location, value) |> # keep only required columns
+  dplyr::filter(location == "48",
+                date >= as.Date("2023-09-23"), date <= as.Date("2024-05-01")) |>
+  dplyr::rename(observation = value) |>
+  dplyr::arrange(date)
+```
+
+```{r, echo=FALSE}
+knitr::kable(target_data[1:5, ], digits = 2,
+             caption = "Five rows of the `target_data` object read in above.
+             The observation column shows the number of hospital admissions
+             due to influenza reported in Texas (location 48) on the week
+             ending on each date.")
+```
+
+\clearpage 
+
+## Visualize forecasts
+
+We can use the hubVis package to create a single graphic that overlays the forecasts from the five models and three forecast dates with the observed data.
+
+```{r, fig.cap = "Forecasts of hospital admissions in Texas due to influenza in the 2023-2024 respiratory virus season. Predictions from five models are shown along with observed data (line with dots for specific observations). Each model is shown in a different color, with the 80% prediction intervals shown as a shaded region.", fig.width = 9, fig.height = 6}  
+flusight_data |>
+  hubVis::plot_step_ahead_model_output(
+    target_data,
+    x_col_name = "target_end_date",
+    use_median_as_point = TRUE,
+    group = "reference_date",
+    interactive = FALSE,
+    fill_transparency = .1,
+    intervals = 0.8
+  ) +
+  ggplot2::theme(legend.position = "bottom") +
+  ggplot2::ylab("hospital admissions due to influenza")
+```
+
+\clearpage 
+
+# Build ensemble and visualize
+
+Using the model output data collected above, we can pass these data to the `hubEnsembles::simple_ensemble()` function to build an ensemble that takes the mean of predicted values from each model at each quantile level. This new forecast output is then added to the existing forecasts and visualized all together. The hubEnsembles package supports a range of aggregation approaches including a linear opinion pool (probability density averaging) or using the mean or median of quantiles.[@shandross_multi-model_2025] Model weights can be specified for the various approaches, allowing for some models to influence the output more strongly.
+
+```{r, fig.cap = "Forecasts of hospital admissions in Texas due to influenza in the 2023-2024 respiratory virus season. Predictions from five submitted models and an ensemble are shown along with observed data. The observed data are shown as a line with dots for specific weekly observations. Each individual model is shown in orange and the mean ensemble is shown in black, with the 80% prediction intervals shown as shaded regions."}
```

Comment:
```suggestion
```{r, fig.cap = "Forecasts of hospital admissions in Texas due to influenza in the 2023-2024 respiratory virus season. Predictions from five submitted models and an ensemble are shown along with observed data. The observed data are shown as a line with dots for specific weekly observations. Each individual model is shown in gray and the mean ensemble is shown in orange, with the 80% prediction intervals shown as shaded regions."}
```
(following @eahowerton  suggestion)